integration/unplugin/README.md

```diff
@@ -84,8 +84,12 @@ interface PluginOptions {
 }
 ```
 
-- `dts`: Whether to generate `.d.ts` type definition files from the Civet source, which is useful for building libraries. Default: `false`
+- `emitDeclaration`: Whether to generate `.d.ts` type definition files from the Civet source, which is useful for building libraries. Default: `false`
 - `typecheck`: Whether to run type checking on the generated code. Default: `false`.
 - `outputExtension`: Output filename extension to use. Default: `.civet.tsx`, or uses `.civet.jsx` if `js` is `true`.
-- `js`: Whether to transpile to JS or TS. Default: `false`.
+- `ts`: Mode of transpilation of TS->JS source code. Default: `civet`. Options:
```

Comment:
I'm also not sure what the best default is... maybe an `auto` mode that tries to `import('esbuild')`, and on failure, tries to `import('typescript')`, and on failure, uses Civet's transpiler? Alternatively, we could try to modify Civet's transpiler to emit a warning if it bumps into not-yet-implemented functionality, and try to use it first, and only on failure try the other systems. (But that would be another PR.)

Also the strings (`"civet"` etc.) should get quotes. Actually so are e.g. `".civet.tsx"`.

Reply:
Maybe we can use civet's transpiler by default, and also show a warning when it's being used without being explicitly set?